src/arco_era5/resize_zarr.py

```diff
@@ -147,3 +147,33 @@ def resize_zarr_target(target_store: str, end_date: datetime, init_date: str,
         logger.info(f"Consolidation of {target_store} is completed.")
     else:
         logger.info(f"Data is already resized for {target_store}.")
+
+
+def update_zarr_metadata(url: str, time_end: datetime.date, metadata_key: str = '.zmetadata') -> None:
+    try:
+        attrs = {"valid_time_start": "1940-01-01",
+                 "valid_time_end": str(time_end),
+                 "last_updated": str(datetime.datetime.now())
+                 }
+        root_group = zarr.open(url)
+
+        # update zarr_store/.zattrs file.
+        for k,v in attrs.items():
+            root_group.attrs[k] = v
```

Comment:
I believe that if you update `root_group.attrs` all at once it is performed in a single IO operation:
```suggestion
        root_group.attrs.update(attrs)
```

Reply:
Thanks, updated.